src/cljc/rems/text.cljc

```diff
@@ -1,8 +1,10 @@
 (ns rems.text
-  (:require #?(:clj [clj-time.core :as time]
+  (:require [clojure.test :refer [deftest is]]
+            #?(:clj [clj-time.core :as time]
                :cljs [cljs-time.core :as time])
             #?(:clj [clj-time.format :as format]
                :cljs [cljs-time.format :as format])
+            #?(:cljs [cljs-time.coerce :as coerce])
```

Comment:
I like to import these as `time-coerce` etc. these days just because they obviously go together.